lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -239,6 +274,12 @@ def error_code_from(response, success)
           response.try(:[], "payment").try(:[], "transaction_status").try(:[], "code")
         end
       end
+
+      def customer_country(options)
+        if country = options[:country] || (options[:billing_address][:country] if options[:billing_address])
+          country.downcase
```

Comment:
Curious why the downcase? Usually country abbreviations are uppercase.

Reply:
Their documentation uses downcase so I figured better safe than sorry.

Reply:
👍 thanks for the explanation!